### Within each subheading, subsets of content will be ranked 1 - X, with 1 being the highest priority.

## Conent drops from standard array backbones (we will be removing these)

1. ACMG2 2016 gene list (we are not focused on general rare variant / exome content - this frees up 67K variants)
2. Blood phenotype genes (these are mainly for newborn / carrier screens - this frees up an additional 26K variants)

## Mandatory content on both backbones

1. 2K variants for risk profiling from NDD GWAS studies
2. Exisitng ILMN pharmaco-genetic content (Hingorani et al.)
3. HGMD extracted variants (differentially filtered for GSA and MEG arrays)
4. KOL Content from Boniface Mok (Asian risk variants), Hjelm (mito) and JC Corvol (additional pharmaco genetic variants)
5. NDD locus tagging variants
6. General imputation boosters for variants > MAF 1%

## Filtering for GSA

1. Risk variants = 2K
2. Pharmaco-genetic variants = 5K
3. HGMD pull (filtered down)
-Disease pull filtered down to just DM flagged variants with RSIDs = 10K variants
-Gene based pull from Genomics England panels filtered down to just DM flagged variants with RSIDs = XK variants
4. KOL content = 2K variants
5. NDD locus tagging variants = 50% of remainder bead types
6. General imputation booster = other 50% of remainder bead types

## Filtering for MEG
1. Risk variants = 2K
2. Pharmaco-genetic variants = 5K
3. HGMD pull (filtered down)
-Disease pull filtered down to just DM flagged variants = 25K variants
-Gene based pull from Genomics England panels filtered down to just DM flagged variants = XK variants
4. KOL content = 2K variants
5. NDD locus tagging variants = 50% of remainder bead types
6. General imputation booster = other 50% of remainder bead types

